.circleci/config.yml

```diff
@@ -0,0 +1,148 @@
+            - run:
+                name: Release build llvm
+                command: ./ci/run_commands.sh
+                environment:
+                    CMAKE_BUILD_TYPE: Release
+
+    # To set up the centos-7 environment:
+    #  - yum install centos-release-scl devtoolset-8 gmp-devel libedit-devel
+    #  - scl enable devtoolset-8 bash
+    build-starexec:
+        docker:
+            - image: centos:7
+              auth:
+                username: mydockerhub-user
+                password: $DOCKERHUB_PASSWORD
+              environment:
+                CMAKE_BUILD_TYPE: Release
+                FLAGS: -Wall -Wextra -Werror
+
+        steps:
+            - run:
+                name: Install the environment
+                command: |
+                    yum -y install centos-release-scl
+                    yum -y install devtoolset-8
+                    yum -y install gmp-devel
+                    yum -y install gmp-static
+                    yum -y install libedit-devel
+                    yum -y install bison
+                    yum -y install git
+                    yum -y install wget
+                    yum -y install rh-python38
+                    yum -y install python3-pip
+                    yum -y install zlib-devel
+                    pip3 install wheel
+                    echo "pip3 install pyinstaller" |scl enable devtoolset-8 bash
+                    yum -y install Cython
+                    yum -y install patch
+                    yum -y install bzip2
+                    yum -y install glibc-static
+                    yum -y install libstdc++-static
+            - run:
+                name: Set up newer version of cmake
+                command: pip3 install cmake
+            - run:
+                name: Set up newer version of flex
+                command: |
+                    cd ~
+                    wget --no-verbose "https://github.com/westes/flex/files/981163/flex-2.6.4.tar.gz"
+                    tar -xvzf flex-2.6.4.tar.gz
+                    cd flex-2.6.4
+                    echo "./configure --prefix=/usr/local; make -j4; make install" | scl enable devtoolset-8 bash
+                    cd ..
+                    rm -rf flex-*
+                    flex --version
+            - checkout
+            - run:
+                name: Set up model validation environment
+                command: echo 'bash ./regression_models/bin/setup-env.sh' |scl enable devtoolset-8 bash
+            - run:
+                name: Debug build gcc under devtoolset-8
+                command: |
+                    cat ./ci/run_commands.sh |scl enable devtoolset-8 bash
+                environment:
+                    CMAKE_BUILD_TYPE: Debug
+            - run:
+                name: Release build gcc under devtoolset-8
+                command: |
+                    cat ./ci/build_maximally_static.sh |scl enable devtoolset-8 bash
```

Comment:
This script is missing now. Maybe it would make sense to have it as well?

Reply:
I have added the script, but this feature was added to OpenSMT after the last release, so we can enable it only after the next release of OpenSMT.